### ISSUE TITLE
Account-Unstyled is empty?

### ISSUE THREAD

**Issue by SachaG** [Meteor-Alt/accounts#4]

**Comment by emmostrom** [comment]
Yes the form default state is empty and to show the log-in form you have to set the state to logIn as you stated.

LogInButtons doesn't exist in un-styled as that is a convenience wrapper in the styled package for AccountStatus and AccountForm to make it easier to switch from the blaze version.

**Comment by SachaG** [comment]
That seems a little counter-intuitive though? Why would the default state be empty?

Also, I wasn't actually able to set the state. Where are you supposed to call `AltAccounts.currentState('logIn')`?

**Comment by emmostrom** [comment]
This is designed to match up with the MDG Blaze version of the accounts package.  The AccountForm is a placeholder for the state-machine to display the various forms depending on what the current state is.  

When a page is encountered for the first time the state is loggedOut.  The default for loggedOut is to show nothing in the AccountForm.

the call can be made from anywhere... usually from a button click when the user wants to log in.
